level2/elb.tf

```diff
@@ -24,12 +24,12 @@ resource "aws_security_group" "alb-sg" {
   }
 }
 
-resource "aws_lb" "public-alb" {
-  name               = "public-alb"
+resource "aws_lb" "private-alb" {
```

Comment:
For the load balancer it should be associated with the public subnet as it will be responsible for routing traffic from the internet to the application managed by the autoscaling group